roles/ipareplica/tasks/install.yml

```diff
@@ -102,6 +102,10 @@
       ipaclient_hostname: "{{ result_ipareplica_test.hostname }}"
       ipaclient_no_ntp: "{{ result_ipareplica_test.ipa_python_version
                             < 40690 }}"
+      ipaclient_ntp_servers:
+        "{{ omit if ipaclient_no_ntp else result_ipaserver_test.ntp_servers }}"
+      ipaclient_ntp_pool:
+        "{{ omit if ipaclient_no_ntp else result_ipaserver_test.ntp_pool }}"
```

Comment:
`result_ipaserver_test` are undefined here, it should be `result_ipareplica_test`.

Reply:
Fixed